portal/portal-render-engine-impl/impl/src/webapp/vm/morpheus/snippets/roleSwitch-snippet.vm

```diff
@@ -8,7 +8,6 @@
     #else
         <a id="roleSwitchDropDownToggle" href="javascript:void('0')" aria-hidden='true' aria-haspopup="true" title='${rloader.rs_menuTooltip}'>
             <i class="fa fa-user-secret" aria-hidden="true"></i>
-            <span class='skip'>${rloader.rs_toggleRoleSwitcher}</span>
```

Comment:
I think it would have been along the lines of "Show View options"!  Happy to change `rs_menuTooltip` to be something more helpful.